Dockerfile

```diff
@@ -41,5 +43,22 @@ COPY --from=build /build/hydra-tui.closure hydra-tui.closure
 
 RUN nix-store --import < hydra-tui.closure && nix-env -i $(cat hydra-tui.drv)
 
+
 STOPSIGNAL SIGINT
 ENTRYPOINT ["hydra-tui"]
+
+# ------------------------------------------------------------------- HYDRAW
+
+FROM nixos/nix:2.3.11 as hydraw
+
+COPY --from=build /build/hydraw.drv hydraw.drv
+COPY --from=build /build/hydraw.closure hydraw.closure
+COPY --from=build /build/hydraw/index.html index.html
+COPY --from=build /build/hydraw/bundle.js bundle.js
+COPY --from=build /build/hydraw/style.css style.css
+COPY --from=build /build/hydraw/logo.png logo.png
+
+RUN nix-store --import < hydraw.closure && nix-env -i $(cat hydraw.drv)
+
+STOPSIGNAL SIGINT
+ENTRYPOINT ["hydraw"]
```

Comment:
would not be better to keep a new DockerFile just for hydraw?
I think would be best to keep this only relevant for running a hydra node, not an application.
wdyt?

Reply:
This re-uses the `build` stage and AFAIK this kind of re-use is not possible across multiple `Dockerfile`s?

Reply:
I don't think it is, no. I suspect if your Dockerfiles are identical you would reuse the layers, but not even sure.